package.json

```diff
@@ -36,8 +36,8 @@
     "When bumping make sure to update the version of react, i.e:  https://github.com/facebook/react-native/blob/v0.56.0/package.json"
   ],
   "peerDependencies": {
-    "react": ">=16.4.1",
-    "react-native": ">=0.56.0"
+    "react": ">=17.0.0",
```

Comment:
Should this be 18 as well?

Reply:
I was actually first using React 17 because I was having issues with 18. And when I resolved them I upgraded but I thought we can leave it a bit looser since it works. I don't have a strong opinion we can also put here 18, officially 0.70 ships with 18 anyway.

Reply:
Makes sense, since the new arch works with 17